downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc

```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
 
 .Procedure
 . Navigate to the `ansible-automation-platform` realm on your SSO client.
-. Under the _Manage_ section on the side navigation bar, click *Users*.
+. Under the _Manage_ section on the side navigation bar, click btn:[Users].
 . Select a user from the list by clicking their ID.
-. Click the *Role Mappings* tab.
+. Click the btn:[Role Mappings] tab.
 . Using the dropdown menu under _Client Roles_, select *automation-hub*.
-. Click *hubadmin* from the _Available Roles_ field, then click *Add selected >*.
+. Click btn:[hubadmin] from the _Available Roles_ field, then click btn:[Add selected >].
```

Comment:
I don't think hubadmin is a button but rather a field selection so just bold with an * on either side.